tests/model_registry/conftest.py

```diff
@@ -190,15 +190,25 @@ def model_registry_instance_oauth_proxy(
 
 @pytest.fixture(scope="class")
 def model_registry_mysql_config(
-    model_registry_db_deployment: Deployment, model_registry_db_secret: Secret
+    request: FixtureRequest,
+    model_registry_db_deployment: Deployment,
+    model_registry_db_secret: Secret,
 ) -> dict[str, Any]:
+    """
+    Fixture to build the MySQL config dictionary for Model Registry.
+    Expects request.param to be a dict. If 'ssl_ca' is not present, it defaults to None.
+    """
+    param = request.param if hasattr(request, "param") else {}
+    ssl_ca = param.get("ssl_ca", None)
+    port = param.get("port", 3306)
```

Comment:
you don't need to save these. You can directly do the param.get() to the line below, where they are used.